src/org/loklak/api/search/SearchServlet.java

```diff
@@ -63,7 +63,7 @@
     private final static String SEARCH_MAX_LOCALHOST_COUNT_NAME = "search.count.max.localhost";
 
     private final static int SEARCH_CACHE_THREASHOLD_TIME = 3000;
-    
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.